pkgs/development/python-modules/filedate/default.nix

```
@@ -0,0 +1,59 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  python-dateutil,
+  setuptools-scm,
+}:
+buildPythonPackage rec {
+  pname = "filedate";
+  version = "3.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "kubinka0505";
+    repo = "filedate";
+    rev = version;
+    hash = "sha256-HvuGP+QlUlfAUfFmaVVvtPHGdrbWVxghQipnqTTvAQc=";
+  };
+
+  unpackPhase = ''
+    cp -r $src/Files/* .
+    cp -r $src/ReadMe.md .
+  '';
+
+  # The repo stores everything in "src" and uses setup.py to move "src" ->
+  # "filedate" before calling setup() and then tries to rename "filedate" back
+  # to "src" after.
+  postPatch = ''
+    mv src filedate
+    substituteInPlace setup.py \
+      --replace-fail "__title__ = os.path.basename(os.path.dirname(os.path.dirname(__file__)))" '__title__ = "filedate"'
+    substituteInPlace setup.py \
+      --replace-fail "cleanup = True" "cleanup = False"
+
+    # Read the ReadMe.md in our patched location
+    substituteInPlace setup.py \
+      --replace-fail 'desc = open("../ReadMe.md", encoding = "UTF-8").read().rstrip()' 'desc = open("ReadMe.md", encoding = "UTF-8").read().rstrip()'
+
+    # Disable renaming "filedate" dir back to "src"
+    substituteInPlace setup.py \
+      --replace-fail "if os.path.exists(__title__):" ""
+    substituteInPlace setup.py \
+      --replace-fail "	os.rename(__title__, directory)" ""
+  '';
+
+  build-system = [ setuptools-scm ];
```

```suggestion
  build-system = [ setuptools ];
```
What we want here is `setuptools`, not `setuptools-scm`.

---

Updated